azdev/operations/setup.py

```diff
@@ -128,6 +141,11 @@ def _install_cli(cli_path, deps=None):
         pip_cmd("install -r {}/src/azure-cli/{}".format(cli_path, req_file),
                 "Installing `{}`...".format(req_file))
 
+    # Ensure that the site package's azure/__init__.py has the old style namespace
+    # package declaration by installing the old namespace package
+    pip_cmd("install -q -I azure-nspkg==1.0.0", "Installing `azure-nspkg`...")
+    pip_cmd("install -q -I azure-mgmt-nspkg==1.0.0", "Installing `azure-mgmt-nspkg`...")
```

Comment:
```suggestion
```
Azure CLI won't need to install these 2 packages for Python namespace correction.